assets/js/phoenix_live_view/live_socket.js

```diff
@@ -692,6 +690,14 @@ export default class LiveSocket {
     }, false)
   }
 
+  maybeScroll(scroll) {
+    if(typeof(scroll) === "number"){
+      setTimeout(() => {
+        window.scrollTo(0, scroll)
+      }, 0) // the body needs to render before we scroll.
```

Comment:
This should probably use requestAnimationFrame instead of a timeout

Reply:
I was not sure if there was a reason why it was a timeout in the first place, so I left that unchanged. But I guess I can change it to requestAnimationFrame.